src/qt/rpcconsole.cpp

```diff
@@ -383,10 +384,13 @@ bool RPCConsole::RPCParseCommandLine(interfaces::Node* node, std::string &strRes
     }
 }
 
-void RPCExecutor::request(const QString &command, const WalletModel* wallet_model)
+void RPCExecutor::request(QString command, const WalletModel* wallet_model)
 {
     try
     {
+        command.replace(QRegExp("@best"), "getbestblockhash()");
+        command.replace(QRegExp("@([\\d]+)"), "getblockhash(\\1)");
```

Comment:
Would it be better to have "\b@best\b" (and likewise for digits) so you don't replace "bob@bestdomainever.com" in a label or similar?